kombu/utils/limits.py

```diff
@@ -21,32 +21,21 @@ class TokenBucket:
         section of any multithreaded code.
     """
 
-    #: The rate in tokens/second that the bucket will be refilled.
-    fill_rate = None
+    def __init__(
+        self,
+        fill_rate: Union[str, int, float],
+        capacity: int = 1
+    ) -> None:
+        #: Maximum number of tokens in the bucket.
+        self.capacity = self._tokens = float(capacity)
 
-    #: Maximum number of tokens in the bucket.
-    capacity = 1
-
-    #: Timestamp of the last time a token was taken out of the bucket.
-    timestamp = None
```

Comment:
These three class variables would always be overwritten in the `__init__`, so they seem redundant to me. I could be wrong, but there are no classmethods or staticmethods or anything else to suggest this is code we want to keep.